.github/workflows/operator-ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - name: Test operator running in cluster
         working-directory: operator
         run: |
-          eval $(minikube -p minikube docker-env) 
+          eval $(minikube -p minikube docker-env)
```

Comment:
@andreaTP no big deal, but I'm curious. Is this change necessary?

Reply:
no, it's not strictly necessary, just good practice to remove trailing white spaces.